src/map/status.cpp

```diff
@@ -12705,7 +12705,8 @@ int status_change_timer(int tid, unsigned int tick, int id, intptr_t data)
 			sc->opt1 = OPT1_STONE;
 			clif_changeoption(bl);
 			sc_timer_next(min(sce->val4, interval) + tick, status_change_timer, bl->id, data);
-			sce->val4 -= interval; //Remaining time
+			if (sce) // check due to sc_timer_next
```

Comment:
How could that be possible?
sc and sce obviously are "available" because they are accessed and checked a few lines above?
https://github.com/rathena/rathena/pull/2695/files#diff-acba7818fb777457f5cbbcea9293f9c4R12618
If we do not modify the "type" variable this should be unreachable code.

Reply:
@Lemongrass3110 The crash is happening because of the ability 'VACUUM EXTREME', if you like I can send you the core dump by PM in the discord or forum.